cmd/goal/examples/justfile

```diff
@@ -0,0 +1,274 @@
+set export
```

Comment:
@tzaffi A few thoughts around `justfile` usage:
* The tool is new to me - thanks for introducing.  I can imagine use cases a `Justfile` is more expressive and/or simpler to put together than a shell/Python script.
* For the PR's purposes, I favor removing `Justfile` and focusing on the docs.  I'm flexible here and if folks feel otherwise, I'm open to discussion.  Rationale:
  * The shown functionality covers some general usage + usage specific to on-going work (box generation).  If we kept all the usage, I imagine a more thorough review taking place.  For example, I'm concerned embedding a Python script into `Justfile` will be difficult to maintain.  
  * On the other hand, I realize the functionality exists because it's helping on-going work.  Out of band to the PR, I think it'd be worthwhile to consider if/where the functionality makes sense to retain.  I can imagine some form of `e2e_sub` test or other varieties.  Though I'd probably first need to learn more about usage.

Reply:
Makes sense to me. I'll remove the file on my next commit.

Reply:
For posterity, I created this [gist](https://gist.github.com/tzaffi/d57018cb826f4766f8f4ce2ba4dbaec8).

Reply:
resolving since I've removed the justfile